Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -5717,6 +5671,55 @@ class BBQJIT {
         return truncTrapping(OpType::I64TruncUF64, operand, result, Types::I64, Types::F64);
     }
 
+    // References
+
+    PartialResult WARN_UNUSED_RETURN addRefIsNull(Value operand, Value& result)
+    {
+        EMIT_UNARY(
+            "RefIsNull", TypeKind::I32,
+            BLOCK(Value::fromI32(operand.asRef() == JSValue::encode(jsNull()))),
+            BLOCK(
+                ASSERT(JSValue::encode(jsNull()) >= 0 && JSValue::encode(jsNull()) <= INT32_MAX);
```

Comment:
Can we make `JSValue::encode` and `jsNull()` constexpr and turn this into a `static_assert` instead?  Maybe can do that in a subsequent patch.

Reply:
Unfortunately, it is union access, and we cannot make it constexpr.